interfaces/webappsec-secure-contexts.idl

```diff
@@ -0,0 +1,7 @@
+// GENERATED CONTENT - DO NOT EDIT
```

Comment:
Can you update from https://github.com/tidoust/reffy-reports/blob/master/whatwg/idl/secure-contexts.idl? That'll also change the filename to match the existing directory name.

Reply:
Done.